### PR TITLE
Update contributing-model-relationships.md

### PR DESCRIPTION
**Pull request by humblenginr**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5ebf37e29cc3d6884f3f21150dffcef1264b84fe at: https://6690136ce471f914b33fb6c4--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-model-relationships.md [hunk 1]
```diff
@@ -330,6 +329,8 @@ Each policy has a set of evaluation rules defined and the `evaluationQuery` attr
 1. To configure a relationship to be applied across all versions of a particular model, ensure the `version` property for those relationships is set to `*`, to limit the relationships to a specific version of a model, specify the correct model version.
 1. Specify `version` property as a regex to ensure relationships are applied to a subset of versions of a model.
 
+{% include alert.html title="Caution While Defining Relationships" type="warning" content="Relationships are not imported directly and they have to be packaged inside a model. Use caution when defining relationships that span models." %}
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-model-relationships.md:332]:
```suggestion
{% include alert.html title="Gratuitous Scoping of Relationships" type="warning" content="Relationships are not imported directly and they have to be packaged inside a model. Use caution when defining relationships that span models." %}
```

Comment (leecalcote) [reply docs/pages/project/contributing/contributing-model-relationships.md:332]: "Relationships are not imported directly and they have to be packaged inside a model." There is some relevance of this information to the warning, but it's not clear what that is given this phrasing and lack of explanation. Please revisit.

Comment (leecalcote) [reply docs/pages/project/contributing/contributing-model-relationships.md:332]: "Use caution when defining relationships that span models." Great. Why? Where's the explanation here?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by Ashparshp** [comment]
@humblenginr Could we discuss this issue during the Meshery dev meeting call? Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by leecalcote** [comment]
@humblenginr, thoughts on the feedback offered? Please accept or reject each point of feedback.

**Comment by l5io** [comment]
🚀 Preview for commit ba04c00016e015912a2d84ff137ceba064db3a10 at: https://669a5eabb2a59436b89e90e3--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-model-relationships.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-model-relationships.md]: This feedback has not been addressed.

**Review by leecalcote — CHANGES_REQUESTED** [review]
The current warning is insufficiently introduced to the reader, leaving them no better off in terms of the objective of this warning.

**Comment by humblenginr** [comment]
Updated the warning

**Comment by l5io** [comment]
🚀 Preview for commit 0ad4eccf04f82d17e85055bf635604af7a3e4488 at: https://669b21d028818da1854f9ebe--meshery-docs-preview.netlify.app

**Pull request closed without merging**